app/javascript/email_pension/EmailPensionView.js

```diff
@@ -245,8 +261,12 @@ export const mapDispatchToProps = (dispatch: Dispatch<*>) => ({
     dispatch(changeName(name));
   },
   changeEmail: (email: string) => dispatch(changeEmail(email)),
+  changeConsent: (consented: boolean) => dispatch(changeConsented(consented)),
```

Comment:
This becomes unnecessary since the ConsentComponent is already tracking that state change. You can use `mapStateToProps` to get the value from state.